code/modules/admin/player_panel.dm

```diff
@@ -492,7 +493,7 @@
 			dat += check_role_table("Ninjas", ticker.mode.ninjas)*/
 
 		if(SSticker.mode.cult.len)
-			dat += check_role_table("Cultists", SSticker.mode.cult, 0)
+			dat += check_role_table("Cultists", SSticker.mode.cult)
```

Comment:
This means the objectives of the cultists would individually be displayed, right? So if we had 10 cultists, the same objectives would be printed 10 times, right?

Reply:
It just shows the "Show Objectives" button